lib/cocoapods-spm/def/spm_package.rb

```diff
@@ -55,6 +55,10 @@ def use_default_xcode_linking?
         @linking_opts.fetch(:use_default_xcode_linking, false)
       end
 
+      def available_in_top_level_target?
+        @linking_opts.fetch(:available_in_top_level_target, true)
```

Comment:
Should this be
- under linking_opts like now? And maybe renamed to "link_to_top_level_target" in then?
- directly inside options?

And is top_level_target an ok name or should it be aggregate target?

Reply:
Under `linking_opts` is fine to me.
Im good with top level target, or even main target. aggregate target sounds a bit misleading as it usually refers to the `Pods-App` target which we dont linking libraries & frameworks to